packages/editor/src/client/blocks/richTextBlock/richTextBlock.tsx

```diff
@@ -141,6 +168,9 @@ export const RichTextBlock = memo(function RichTextBlock({
         onBlur={() => {
           setLocation(editor.selection)
         }}
+        onKeyDown={e => {
+          if (e.ctrlKey) activateHotkey(e)
```

Comment:
Can we have e.preventDefault() here? what do you think?

Reply:
Unfortunately not because it would disable typing inside the editor. Even placing it inside the activateHotkey method, but outside of the switch statement, would disable other shortcuts like copy/paste and I haven't thought of any obvious way to avoid the repetitiveness.